src/time_open/main.d

```diff
@@ -0,0 +1,29 @@
+/*******************************************************************************
+
+    Main file
+
+    Copyright:
+        Copyright (c) 2019 dunnhumby Germany GmbH. All rights reserved.
+
+    License:
+        Boost Software License Version 1.0. See LICENSE.txt for details.
+
+*******************************************************************************/
+
+module time_open.main;
+
+
+/*******************************************************************************
+
+    Main function
+
+    Params:
+        args = command line arguments
+
+*******************************************************************************/
+
+version(unittest) {} else
```

Comment:
This is probably a bad idea. When `make unittest` is done, there will be 2 `main` and the linker will complain.

Reply:
I'm confused I just copied from `src/autopr/main.d`. Why will there be two `main`s? Doesn't this only add the main, when you are not in a unittest.

Reply:
I push code in a fixup commit to add an empty main for `unittest`s. I'm still puzzled by the other tools don't need it.